firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,156 @@ extension String {
         comment: "Question shown to user when tapping an SMS or MailTo link that opens the external app for those.")
 }
 
+// MARK: Enhanced Tracking Protection/Unified Trust Panel
+extension String {
+    public struct EnhancedTrackingProtection {
+        public static let onTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%@ is on guard",
+            comment: "Title for the enhanced tracking protection screen when the user has selected to be protected. The placeholder will have the value of the app name")
+
+        public static let onHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "You’re protected. If we spot something, we’ll let you know.",
+            comment: "Header for the enhanced tracking protection screen when the user has selected to be protected.")
+
+        public static let offTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Off.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "You turned off protections",
+            comment: "Title for the enhanced tracking protection screen when the user has opted out of the feature.")
+
+        public static let offHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Off.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Firefox is off-duty. We suggest turning protections back on.",
+            comment: "Header for the enhanced tracking protection screen when the user has opted out of the feature.")
+
+        public static let onNotSecureTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.NotSecure.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Be careful on this site",
+            comment: "Title for the enhanced tracking protection screen when the user has selected to be protected but the connection is not secure.")
+
+        public static let onNotSecureHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Your connection is not secure.",
+            comment: "Header for the enhanced tracking protection screen when the user has selected to be protected but the connection is not secure.")
+
+        public static let connectionVerifiedByLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Verifier.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Verified by %@",
+            comment: "String to let users know the site verifier, where the placeholder represents the SSL certificate signer.")
```

Comment:
Instead of `String`, should we use `Text` since these are read by the contributors that translate and may not know what a string refers too. Also might be helpful to also mention its on the enhanced tracking protection screen after the user taps on the secure connection details. 

Similar note for the following cases where we use `String`